chain/consensus/filcns/filecoin.go

```diff
@@ -436,15 +436,15 @@ func (filec *FilecoinEC) VerifyWinningPoStProof(ctx context.Context, nv network.
 	return nil
 }
 
-func isValidForSending(act *types.Actor) bool {
+func IsValidForSending(act *types.Actor) bool {
 	if builtin.IsAccountActor(act.Code) {
 		return true
 	}
 
-	// HACK: Allow Eth embryos to send messages
-	if !builtin.IsEmbryo(act.Code) || act.Address == nil || act.Address.Protocol() != address.Delegated {
+	if !builtin.IsEmbryo(act.Code) || act.Address == nil || act.Address.Protocol() != address.Delegated || act.Nonce != 0 {
```

Comment:
Also ethereum accounts? Or are we including that in `IsAccountActor`?

Reply:
Eth Accounts didn't exist yet -- adding now.